### PR TITLE
DAT-56138

### PR DESCRIPTION
**Pull request by makc-dataloop**
https://dataloop.atlassian.net/browse/DAT-56138

### CODE REVIEW

File: src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchJsonEditorDialog.vue [hunk 1]
```diff
@@ -99,7 +99,7 @@
                         :disabled="!newQueryName"
                         outlined
                         style="margin-right: 5px"
-                        @click="saveQuery"
+                        @click="() => saveQuery(false)"
```
Comment (fadiDL) [inline src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchJsonEditorDialog.vue:102]: @makc-dataloop Why is this needed ? the default of this query is false so when you click saveQuery it will have a default value 
wrapping it and explicitly calling shouldnt matter ?

**Review by fadiDL — CHANGES_REQUESTED** [review]

File: src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchJsonEditorDialog.vue (hunk 1 above)
Comment (makc-dataloop) [reply src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchJsonEditorDialog.vue:102]: @fadiDL unfortunately it receives an event for an argument and it is != false

**Review by fadiDL — APPROVED** [review]
